spec/workers/metrics/record_db_and_es_record_counts_worker_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+
+RSpec.describe Metrics::RecordDbAndEsRecordCountsWorker, type: :worker, elasticsearch: true do
+  include_examples "#enqueues_on_correct_queue", "low_priority", [0]
+
+  describe "#perform" do
+    let(:worker) { subject }
+
+    it "Records a matching count" do
+      allow(DatadogStatsClient).to receive(:increment)
+      create(:tag, :search_indexed)
+
+      Search::Tag.refresh_index
+      worker.perform
+
+      tags = { search_class: Search::Tag, db_count: 1, index_count: 1, record_difference: 0, margin_of_error: 0, action: "record_count", record_count: "match" }
```

Comment:
This is just to make the linter happy. Making it more readable takes up several lines and goes over the limit.